### ISSUE TITLE
Get a map view from API

### ISSUE THREAD

**Issue by apachler** [osmandapp/osmand-api-demo#26]
Is it possible to get a map view object from the API to display OBF files in other apps than OsmAnd?

Or is any other way or library existing to load OBF files in other projects?

**Comment by vshcherb** [comment]
There is a method to get map image at least it works for OsmAnd Telegram to display a picture for track

**Comment by apachler** [comment]
OK, but a screenshot ist not enough. I want to load OBF map files in an self made Android app. So i need an API or library to load OBF map files, is that existing?

**Comment by vshcherb** [comment]
I don't think so, there is no API to parse OBF flles, it would be strange of course cause it is better to do within the app itself

**Comment by apachler** [comment]
Is there no library for OBF map file parsing existing? I was not able to find one. Or would it be possible to use some code from OsmAnd? Any hints?
